daemon/create.go

```diff
@@ -144,5 +144,9 @@ func (daemon *Daemon) VolumeCreate(name, driverName string, opts map[string]stri
 	if err != nil {
 		return nil, err
 	}
+
+	if driverName != "" && v.DriverName() != driverName {
```

Comment:
@runcom , yes , because if user don' t give a driver name when create a  volume the driverName will be "" and the volume will be given a default driver name "local", no matter the volume is exist or not, it' s not a conflict case